astra/src/main/java/com/slack/astra/logstore/schema/SchemaAwareLogDocumentBuilderImpl.java

```diff
@@ -381,7 +313,7 @@ public Document fromMessage(Trace.Span message) throws JsonProcessingException {
           doc,
           LogMessage.SystemField.ID.fieldName,
           message.getId().toStringUtf8(),
-          Schema.SchemaFieldType.KEYWORD,
+          Schema.SchemaFieldType.ID,
```

Comment:
this is important. previosuly since `_id` was predefined, we had it wrongly set to KEYWORD which meant all tests were failing.